### PR TITLE
DL-12454-Play3.0 Upgrade- Single Commit with all changes

### PR DESCRIPTION
**Pull request by rameshmandanapu1**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

*Reviewee* (Replace with your name)
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

*Reviewer* (Replace with your name)
 - [ ]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by paulanderson1234** [comment]
There is almost certainly a few things that could be removed here based on recently reviewed changes to ACM and ATED FE upgrades. We may be able to 

- remove TestPhases file if removed from build.sbt
- remove mockito-core, mockito-4-11, and jackson modules as bootstrap comes with a version that may suffice for tests
- remove it dependency as we have found somewhere in IntegrationBase (or similar) it will import wiremock from tomakehurst (therefore rendering the one we have updated as not required)
- need to check if akka.stream is present in logback file as this probably needs updating to org.apache.pekko.stream

**Comment by rameshmandanapu1** [comment]
.settings(inConfig(TemplateTest )(Defaults.testSettings) *) is calling testphases from build.sbt file, Unable to remove.

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -1,39 +1,28 @@
...
-        "com.fasterxml.jackson.module" %% "jackson-module-scala"   % "2.15.3"            % scope,
-        "com.github.tomakehurst"       %  "wiremock-jre8"          % "2.35.1"            % scope
-      )
-    }.test
-  }
-
-  def apply(): Seq[ModuleID] = compile ++ Test()
-}
+  val itDependencies: Seq[ModuleID] = Seq(
+    "org.wiremock" % "wiremock" % "3.4.2" % Test
```
Comment (benryan1790) [inline project/AppDependencies.scala:25]: This needs to be removed, with it in the it tests do not work

File: app/controllers/ApplicationController.scala [hunk 2]
```diff
@@ -38,34 +38,34 @@ class ApplicationController @Inject()(mcc: MessagesControllerComponents,
 
   implicit val ec: ExecutionContext = mcc.executionContext
 
-  def unauthorised(): Action[AnyContent] = Action { implicit request =>
+  def unauthorised: Action[AnyContent] = Action { implicit request =>
     Ok(templateUnauthorised())
   }
 
-  def cancel(): Action[AnyContent] = Action { implicit request =>
+  def cancel: Action[AnyContent] = Action { implicit _ =>
```
Comment (benryan1790) [inline app/controllers/ApplicationController.scala:45]: Why were these changed?

**Review by benryan1790 — CHANGES_REQUESTED** [review]

**Comment by rameshmandanapu1** [comment]
> * .settings(inConfig(TemplateTest )(Defaults.testSettings) *) is calling testphases from build.sbt file, Unable to remove.
> 
>     * getting below error when removing Jackson dependency - com.fasterxml.jackson.databind.JsonMappingException: Scala module 2.14.3 requires Jackson Databind version >= 2.14.0 and < 2.15.0 - Found jackson-databind version 2.16.1

**Pull request merged**